docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,20 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides the trusted CA certificate that defaults to the curated list of well-known CAs by Mozilla.
+
+### httpsPrivateKey
+
+Specifies the private key in PEM format for mTLS authentication.
+**Note: a Private Key SHOULD NOT be specified directly here to avoid losing confidentiality completely.
+Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.**
```

Comment:
```suggestion
**Note: a Private Key SHOULD NOT be specified directly here to avoid losing confidentiality completely.**
Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.
```

Reply:
```suggestion
<!-- prettier-ignore -->
!!! warning
    Do _not_ specify your private key directly here, to avoid losing confidentiality completely.
    Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.
```

This is important, so we should put it in an admonition, so it draws the reader's eye. I've also rewritten the warning a bit. Check if this is correct before applying my suggestion.